lib/client/signalflow/signalflow_client.js

```diff
@@ -1,8 +1,7 @@
-'use strict';
+"use strict";
```

Comment:
Could you keep all those stylistic changes to another commit in this PR?

Reply:
I think I auto added some of these unintentionally, let me revert.